packages/fluentui/react-northstar/src/themes/teams/components/Chat/chatMessageContentStyles.ts

```diff
@@ -0,0 +1,55 @@
+import { ComponentSlotStyleFunction, ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+
+import { ChatMessageContentStylesProps } from '../../../../components/Chat/ChatMessageContent';
+import { ChatMessageVariables } from './chatMessageVariables';
+import { pxToRem } from '../../../../utils';
+
+const getChatMessageVariantStyles: ComponentSlotStyleFunction<ChatMessageContentStylesProps, ChatMessageVariables> = ({
+  props: p,
+  variables: v,
+  theme,
+}) => {
+  return {
+    ...(p.layout === 'refresh' &&
+      p.density === 'comfy' && {
+        color: v.contentColor,
+        wordBreak: 'break-word',
+        wordWrap: 'break-word',
+        '& a': {
+          color: 'inherit',
+          textDecoration: 'underline',
+          wordBreak: 'break-all',
+          '&:hover': { textDecorationStyle: 'double' },
+          '&:focus': { textDecorationStyle: 'double' },
+        },
+        ...(p.failed && {
+          color: theme.siteVariables.colorScheme.default.foreground,
+        }),
+      }),
+
+    ...(p.density === 'comfy' && {
+      ...(p.hasBadge && p.badgePosition === 'end' && { marginRight: pxToRem(4) }),
+    }),
+  };
+};
+
+export const chatMessageContentStyles: ComponentSlotStylesPrepared<
+  ChatMessageContentStylesProps,
+  ChatMessageVariables
+> = {
+  root: (componentStyleFunctionParam): ICSSInJSStyle => {
+    const { props: p, variables: v } = componentStyleFunctionParam;
+    return {
+      color: v.contentColor,
+      display: 'block',
+      '& a': {
+        outline: 'none',
+        color: p.mine ? v.linkColorMine : v.linkColor,
+        ':focus': {
+          textDecoration: 'underline',
+        },
+      },
+      ...getChatMessageVariantStyles(componentStyleFunctionParam),
```

Comment:
Why have this as a function? I only see it used once.

Reply:
I wanted to keep the other layout-specific styles separated. But now looking at it, removing the function makes more sense.